pep-9999.rst

```diff
@@ -0,0 +1,874 @@
+  users for invalid expressions (but still accept any string as `License`).
+
+
+Scope
+=====
+
+This PEP scope is strictly about how we document the license of a package:
+
+- with an improved and structured way to document a license expression, and,
+- by including license texts in a built package.
+
+The metadata specification changes have been designed to have a minimal impact
+and to be backward compatible with v2.1.  They are embracing emerging new ways
+to document licenses that are already in use in some tools (e.g. by adding
+`license_file` field already used in wheel and setuptools) or by some package
+authors (e.g. storing an SPDX license expressions in the existing `license`
+field). 
+
+All the packages that conform with the v2.1 of core metadata specification are
+also conformaing and valid with this v2.2 core metadata specification.
+
+Beyond this metadata specification update, this PEP contains:
+
+- recommendations for package Publishing tools on how to validate the `License`
+  and `Classifier` fields and report informational warnings when a package uses
+  an older, non- structured style of license documentation conventions.
+
+- informational appendixes that contain surveys of how we document license
+  today in Python packages and elsewhere, and a reference Python library to
+  parse, validate and build correct license expressions.
+
+It is the intent of the PEP authors to work closely with tools authors to
+implement to recommended `License` field validation and warnings specified in
+this PEP.
+
+
+What this PEP is not about
+==========================
+
+This PEP and its authors are neutral with regards to using this or that license.
+
+In particular, the SPDX license expression syntax proposed in this PEP provides
+simpler and more expressive conventions to document more accurately any kind of
+license that applies to a Python package, whether under an open source, free or
+libre software license or a proprietary license.
+
+Therefore this PEP makes no recommendation to force using certain licenses or
+using certain license documentation conventions and does not recommend any
+license at all (including any license restriction when uploading to PyPI).
+
+Instead it only enacts in documentation some common and/or improved practices
+already in use, and recommend that Publishing tools should gently nag users with
+informational warnings when they do not follow this PEP recommendations.
+
+This PEP is not about documenting license in code files, even though this is a
+surveyed topic in Appendix.
+
+
+Possible future PEPs
+====================
+
+It is the intention of the authors of this PEP to consider the submission of
+related but separate PEPs in the future such as:
+
+ - making the existing `License` and new `License-File` fields mandatory
+   including stricter enforcement in tools and Pypi publishing.
+
+ - restricting the upload of packages to the public Pypi index to the packaes
```

Comment:
`packaes` -> `packages`

Worth noting: SPDX license list has metadata for whether the license is approved by OSI and FSF.

Reply:
Fixed in 86eb7a88a9ada9e4a753e2a8b9f17eabd4e447a3